docs/source/search/granules.ipynb

```diff
@@ -0,0 +1,404 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Searching by Temporal Filter\n",
+    "\n",
+    "Here we will combine a search from earlier using the Collection Concept ID with a temporal filter to fine tune our results using the `temporal` keyword in our search.\n",
+    "\n",
+    "The temporal keyword takes datetime information in a [specific format](https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html#g-temporal). The date format used is `YYYY-MM-DDThh:mm:ssZ` and temporal search criteria may be either a single date or a date range. If one date is provided then it can be inferred as start or end date. To define a start date and return all granules after the date, put a comma after the date (`YYYY-MM-DDThh:mm:ssZ,`). To define a end date and return all granules prior to the data, put a comma before the date (`,YYYY-MM-DDThh:mm:ssZ`). Lastly, to get a date range provide the start date and end date separated by a comma (`YYYY-MM-DDThh:mm:ssZ,YYYY-MM-DDThh:mm:ssZ`).\n",
+    "\n",
+    "In this example we will search for one month of data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "'Got 500 results'\n"
+     ]
+    }
+   ],
+   "source": [
+    "dateRange = '2018-12-01T00:00:00Z,2018-12-31T23:59:59Z' # specify date range to search for data in Dec. 2018\n",
+    "\n",
+    "results = maap.searchGranule(temporal=dateRange,concept_id=COLLECTIONID,limit=MAXRESULTS,)\n",
+    "pprint(f'Got {len(results)} results')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "'Granule ATL08_20181201001339_09680103_002_01.h5 was acquired on 2018-12-01T00:15:04.914Z'\n"
+     ]
+    }
+   ],
+   "source": [
+    "granuleFilename = results[0]['Granule']['DataGranule']['ProducerGranuleId'] # get the granule file name\n",
+    "granuleDate = results[0]['Granule']['Temporal']['RangeDateTime']['BeginningDateTime'] # get the granule start time\n",
+    "\n",
+    "pprint(f'Granule {granuleFilename} was acquired starting at {granuleDate}',width=100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It looks like the first result correctly matches with the beginning temporal search parameter. Keep in mind that the results are limited to 100 so the final granule returned may not match the end date that was searched for."
```

Comment:
Just noticed that the 100 in this paragraph should be 500.

Reply:
Just committed the change